Tribler/Core/Modules/restapi/events_endpoint.py

```diff
@@ -88,9 +121,9 @@ def render_GET(self, request):
         """
         self.events_request = request
 
-        request.write(json.dumps({"type": "events_start"}))
+        request.write(json.dumps({"type": "events_start"}) + '\n')
```

Comment:
@whirm that's a very good point. Shall I create a separate issue for it and fix it in another PR?